scripts/install/jellyfin.sh

```diff
@@ -40,6 +34,47 @@ if [[ -n $active ]]; then
     fi
 fi
 
+ARCHITECTURE="$(dpkg --print-architecture)"
+BASE_OS="$(awk -F'=' '/^ID=/{ print $NF }' /etc/os-release)"
+
+# Handle some known alternative base OS values with 1-to-1 mappings
+# Use the result as the repository base OS
+case "${BASE_OS}" in
```

Comment:
This case is basically useless aside from setting the _os_codename. If an OS doesn't directly identify itself as Debian or Ubuntu (i.e. during install) then we don't support it.

We certainly don't support linux mint or kde neon

Reply:
I was unaware we don’t support those distros at all. I will be removing them from the scripts.

Reply:
I removed unsupported distros, but I am not aware if _os_codename is set as debian on raspbian. It is necessary to make sure raspbian is set to the same repos as debian, as that is what Jellyfin requires.